### PR TITLE
DP-1.9 - WRR traffic test - Adding required vendor specific queue name change and interface-ref definition

### PR DESCRIPTION
**Pull request by charantejag504**
DP-1.9 - WRR traffic test - Adding required vendor specific queue name change and interface-ref definition
1. Added Queue ID as required by Juniper.
     Other pull request containing such changes: https://github.com/openconfig/featureprofiles/pull/1087
     Related OC model change: https://github.com/openconfig/public/pull/772
2. Included existing deviation ExplicitInterfaceRefDefinition for interface configuration under QoS(classifier and scheduler)

### CODE REVIEW

File: feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go [hunk 1]
```diff
@@ -1171,38 +1171,64 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
...
+		be0: "BE0",
+		be1: "BE1",
+		af1: "AF1",
+		af2: "AF2",
+		af3: "AF3",
+		af4: "AF4",
+		nc1: "NC1",
+	}
+
+	if dut.Vendor() == ondatra.JUNIPER {
```
Comment (greg-dennis) [inline feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go:1188]: Where is the deviation that tracks the need for this change?

**Review by greg-dennis — CHANGES_REQUESTED** [review]

File: feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go (hunk 1 above)
Comment (greg-dennis) [reply feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go:1188]: As stated elsewhere, I received confirmation that we are punting on ironing out this non-uniformity. I am tracking it elsewhere

**Review by greg-dennis — APPROVED** [review]
Approving for Go quality only

File: feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go (hunk 1 above)
Comment (charantejag504) [reply feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go:1188]: same reason as https://github.com/openconfig/featureprofiles/pull/1378, this is not a deviation

**Review by greg-dennis — CHANGES_REQUESTED** [review]
Please update the branch and resolve conflicts

**Comment by charantejag504** [comment]
> Please update the branch and resolve conflicts

Branch is updated and no merge conflict seen

**Review by sachendras — APPROVED** [review]

**Pull request merged**